inference_rules.adoc

```diff
@@ -474,12 +478,17 @@ Fabric and protocol must be reported in the submission metadata. Submission meta
 ==== SUT over the Network and Setup Constraints
 
 * SUT parameters and configuration must be uniquely and specifically named in the submission results.
-* All queries must be transferred over the network, for inference execution at the SUT. All responses must be transferred back over the network. Caching of the queries or responses for further use at the SUT is disallowed.
-* Network interface characteristics must be added to the report, both at the SUT side and at the Loadgen system side, including networking components, and name any other components through which data passes between Loadgen and the SUT for reproducibility.
+* Everything outside the LoadGen node should be considered as part of the SUT, for instance for counting power and latency. As an example, components outside the nodes like switch or load balancer should be considered part of the SUT. 
+* All queries must be transferred over the network, carrying the inference data, for inference execution at the SUT. All responses must be transferred back over the network, carrying the inference responses. 
+* Caching/Storing of the queries and inference data or responses for further use at the SUT is disallowed. It is allowed to cache/store other data like Neural Network weights or Neural Network executable.
+* SUT can do the required pre-processing of the data, e.g. Batching, Padding, processing of the requests (precision, data layout), compression, decompression. SUT can do the required post processing functions e.g. gather, reduction or ArgMax. 
+* Network interface characteristics must be added to the report, both at the SUT side and at the LoadGen system side, including networking components, and name any other components through which data passes between LoadGen and the SUT for reproducibility.
+* It is required to add a system diagram that shows how the components between the LoadGen node and the SUT node are connected and to add text that explains the diagram.
+* For reproducibility, it is required to specify software version of all components, HW configurations, software stacks, dockers and settings.
```

Comment:
(1) missing oxford comma before final dockers and settings (2) dockers, and settings:  settings of what?

Reply:
settings of all components and stacks